smart_open/gcs.py

```diff
@@ -39,17 +40,19 @@
 DEFAULT_BUFFER_SIZE = 256 * 1024
 """Default buffer size for working with GCS"""
 
-_UPLOAD_INCOMPLETE_STATUS_CODE = 308
+_UPLOAD_INCOMPLETE_STATUS_CODES = (308, )
 _UPLOAD_COMPLETE_STATUS_CODES = (200, 201)
 
 
-def _make_range_string(start, stop=None, end=_UNKNOWN_FILE_SIZE):
+def _make_range_string(start, stop=None, end=None):
```

Comment:
Is None as a default arg really cleaner than the actual default?

Reply:
Yeah, I think so. That asterisk is an implementation detail of the `make_range_string` function. You don't want to leak it to callers of the function. It's also more Pythonic to use None as opposed to a constant to indicate the absence of a value.